grommet/grommet
# Allow aria-label to be used where a11yTitle is accepted

#### What does this PR do?
Leverages work started in #5137 but resolves all merge conflicts. Adds tests where appropriate.

#### Where should the reviewer start?

#### What testing has been done on this PR?
Added tests and reviewed snapshots.

#### How should this be manually tested?
Add `aria-label` to a component such as Menu. Should see `aria-label` in the DOM.

#### Any background context you want to provide?

#### What are the relevant issues?

#### Screenshots (if appropriate)

#### Do the grommet docs need to be updated?
No.

#### Should this PR be mentioned in the release notes?
Yes, improved support for `aria-label`

#### Is this change backwards compatible or is it a breaking change?
Backwards compatible.
